src/rcc.rs

```diff
@@ -80,13 +78,10 @@ impl APB1 {
     }
 }
 
-
 impl APB1 {
     /// Set power interface clock (PWREN) bit in RCC_APB1ENR
     pub fn set_pwren(&mut self) {
-        self.enr().modify(|_r, w| {
-            w.pwren().set_bit()
-        })
+        self.enr().modify(|_r, w| w.pwren().set_bit())
```

Comment:
It looks like your commit includes ```cargo fmt``` formatting changes. IMHO it makes sense to keep coding style changes in a separate commit just to simplify review.

Reply:
I'll back these out and update the PR shortly.

Reply:
Done and pushed